src/Spryker/Glue/AppKernel/AppKernelDependencyProvider.php

```diff
@@ -9,6 +9,7 @@
 
 use Spryker\Glue\AppKernel\Dependency\Facade\AppKernelToAppKernelFacadeBridge;
```

Comment:
Bridge is not needed. Spryker\Glue\Kernel\Backend\AbstractFactory has access to getFacade() method.

Reply:
I don't see the facade being accessible. Specific version required?

Reply:
I checked wrong example, sorry. But why Bridge to access the same module Facade?

Reply:
CS Sniffer complains

Reply:
I removed PHPCS does not care (green CI jobs), so not sure which sniffer you mean.

Reply:
The PHPCS would find it with `\Spryker\Sniffs\DependencyProvider\FacadeNotInBridgeReturnedSniff::isFacadeNotInBridgeReturned()` which will not happen because Rector renamed the `$container` variable to `$glueBackendContainer` as the variable names have to match the Class name used.

We have `use Spryker\Glue\Kernel\Backend\Container as GlueBackendContainer;` (not sure why) but because of this, the Sniffer doesn't complain anymore.

I'd recommend reverting your changes back to what it was.

Reply:
again, it's not SCOS module and should not follow a stupid rule (creation a Bridge to use in the same module) and you just found a nice workaround to avoid it. So my opinion it should not be reverted, but this rule disabled for this repo.

Reply:
From my point of view, it doesn't matter if it is in SCOS or not. It will also be part of the "open-the-platform" initiative and even if we find this rule problematic we should not change it without confirmation by someone else. We have this rule and if we want to change it we either need support from someone else or make an RFC.

Reply:
There is no use case in the Core for this.
It's always two modules: the main module and BackendApi module (Product and ProductsBackendApi).
AppKernel case is another case:  BackendApi and main Facade are in one module.